### DISCUSSION TITLE
No access to my deployed projects and domains after rearranging my git provider accounts

### DISCUSSION THREAD

**Discussion by maximilian-V** [vercel/community#4691] [Help] [answered]
### Question

Hello, 

Today I created a new account and added my git profiles to the new account, thinking it would also transfer my projects. Shortly after, I noticed that I can not see my projects in my new accounts dashboard. The deployments are still shown as deployed in mit Github repo. When I redeploy the repo from my git account that contains the project, a new production environment is created. Also, I can no longer access my domains even though they should be connected to my account. I would like to know how i can get access to my old dashboard again. 

 Does anyone know how I can fix these issues? I thought the connection to my e-mail would still keep all my projects in place.

**Comment by amyegan** [comment, marked as answer]
Hi @maximilian-V! Projects belong to your account or team. So they are not automatically transferred when you switch which Vercel account is linked to your Git remote account. 

You can learn more about transferring a project here: https://vercel.com/docs/projects/overview#transferring-a-project
